.github/workflows/build-and-test.yml

```diff
@@ -3,6 +3,6 @@ on:
   pull_request:
 jobs:
   build:
-    uses: 'flowfuse/github-actions-workflows/.github/workflows/build_node_package.yml@132a77418c21639c18318ed6a687e28caa7f4ce8'
+    uses: 'flowfuse/github-actions-workflows/.github/workflows/build_node_package.yml@v0.1.0'
```

Comment:
should these now be 0.1.1 (once tagged)?

Reply:
Bumped to 0.1.1.